xml/pop_banners.xml

```diff
@@ -217,7 +217,58 @@ matched against these patterns to fingerprint POP3 servers.
     <param pos="0" name="service.product" value="Courier POP"/>
   </fingerprint>
   <!--
-
+<fingerprint pattern= "^CMailServer ([\d\.]+) POP3 Service Ready$">
+    <description>matches to CMailServer</description>
+    <example>CMailServer 5.0.0 POP3 Service Ready</example>
+	<param pos="0" name="service.product" value="CMailServer"/>
+	<param pos="0" name="service.vendor" value="Youngzsoft"/>
+	<param pos="0" name="os.family" value="Windows" />
```

Comment:
Whitespace - here and elsewhere in the PR <param> should be at the same indention as <example> and <description>.